plugins/carbon_black_cloud/unit_test/test_monitor_alerts.py

```diff
@@ -294,7 +294,7 @@ def test_http_exceptions(
 
         self.assertEqual(expected_status_code, status_code)
         self.assertEqual(num_logs, len(response))
-        self.assertFalse(has_more_pages)
+        self.assertEqual(has_more_pages, status_code == 200)
```

Comment:
can this be updated to `assertTrue`? just a bit strange we're asserting the status code on line 295 and then again in here to assert has_more_pages?

Reply:
No this should only be true in the case status_code == 200

Reply:
Oh I hadn't expanded the PR further to see we have a bunch of other scenarios being tested. Makes sense!